app/models/collection.rb

```diff
@@ -5,12 +5,29 @@ class Collection < ActiveFedora::Base
   include ::Hyrax::CollectionBehavior
   # You can replace these metadata if they're not suitable
 
+  property :abstract, predicate: ::RDF::Vocab::DC.abstract,
```

Comment:
Most of the changes here is 
1) removes hyrax basic metadata
2) re-ogranizes the properties to by in alphabetical order